CLAUDE.md

```
@@ -0,0 +1,126 @@
+npm run build        # Production build
+npm run lint         # ESLint
+npm run test         # Vitest (watch mode)
+npm run test:run     # Vitest (single run)
+npm run routes       # Regenerate TanStack Router tree (tsr generate)
+```
+
+## Architecture
+
+Browser-based multi-track video editor. React 19 + TypeScript + Vite.
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)